Scripts/pilconvert.py

```diff
@@ -46,7 +46,7 @@ def usage():
     print(v)
     sys.exit(1)
 
-format = None
+outputFormat = None
```

Comment:
Thanks for the link, I clearly wasn't aware of this convention. I've fixed this now.